eng/dashboardpack/dashboardpack.csproj

```diff
@@ -35,6 +35,9 @@
       <None Include="@(_PublishItems)" Pack="true" PackagePath="tools/" />
     </ItemGroup>
 
+    <MakeDir Directories="$(DashboardPublishedArtifactsOutputPath)/$(DashboardRuntime)" />
+    <ZipDirectory SourceDirectory="$(DotNetOutputPath)Aspire.Dashboard/$(Configuration)/$(TargetFramework)/$(DashboardRuntime)/publish" DestinationFile="$(DashboardPublishedArtifactsOutputPath)/$(DashboardRuntime)/aspire-dashboard-$(DashboardRuntime).zip" />
+
     <!-- Throw an error if _PublishItems is empty. -->
     <Error Condition="'@(_PublishItems)' == ''" Text="No files were found to pack. Ensure that the project being packed has a publish target defined." />
```

Comment:
Just curious - why not have this in a target in the Dashboard project itself? we would have access to the publish path also without having to construct it manually.

Reply:
This is a valid point, but that would make it so that we would generate zips every time we are publishing the individual dashboard project which won't always be desirable. Given this whole project's purpose is mainly to get things ready for the workload publishing, and the fact that getting  the zips is mostly a concern only for official builds, I think I'm leaning to keep this here for now. If you feel strongly on moving this, I'm happy to take care of it as a follow up PR.

Reply:
Sounds fair 👍